### PR TITLE
USHIFT-506: Remove pkg/assets/bindata.go

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Review by stlaz — CHANGES_REQUESTED** [review]
you'll want to fix unit tests which still seem to contain references to the assets.

`pkg/components/render_test.go`, for example

File: pkg/components/render_test.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 	"text/template"
 
+	embedded "github.com/openshift/microshift/assets"
```
Comment (stlaz) [inline pkg/components/render_test.go:12]: I would rather see:
1. remove the bindata.go
2. rename the package in `microshift/assets/embed.go` back to `assets` and change this wherever needed
3. profit :slightly_smiling_face:

**Review by stlaz — CHANGES_REQUESTED** [review]

File: pkg/components/render_test.go (hunk 1 above)
Comment (pmtk) [reply pkg/components/render_test.go:12]: Problem is we already have `pkg/assets` package which imports `assets/`

Comment (stlaz) [reply pkg/components/render_test.go:12]: I see... I think I'll try to fix this afterwards, lets' stick with what you've got at this moment, then :slightly_smiling_face:

**Pull request merged**